spec/services/measurements_creator_spec.rb

```diff
@@ -0,0 +1,60 @@
+require "spec_helper"
+
+describe MeasurementsCreator do
+  let(:measurements_attributes) { [{:longitude=>25.4356212, :latitude=>56.4523456, :time=>"2016-05-11T17:09:02", :milliseconds=>925, :measured_value=>59.15683475380729, :value=>59.15683475380729}] }
+
+  it "creates a mesurement" do
+    stream = create_stream!
```

Comment:
Would the test work if the stream was not persisted? (ie `Stream.new` instead of `Stream.create`). That way we avoid one trip to db and make tests faster

Reply:
Not in the current form. Couse `Steram.build_measurements` uses the database. But maybe this should be a unit test and I should stub all methods in stream class??? But I'm not sure I know how to do this.